starters/solidstart-tanstackquery-tailwind-modules/README.md

```diff
@@ -1,30 +1,87 @@
-# SolidStart
+# SolidStart-TanStackQuery-Tailwind-Modules
 
-Everything you need to build a Solid project, powered by [`solid-start`](https://start.solidjs.com);
+This starter kit features **SolidStart**, combined with **TanStack Query**, and **Tailwind Modules**.
 
-## Creating a project
 
-```bash
-# create a new project in the current directory
-npm init solid@latest
+## Table of Contents
 
-# create a new project in my-app
-npm init solid@latest my-app
-```
+- [Overview](#overview)
+  - [Tech Stack](#tech-stack)
+  - [Included Tooling](#included-tooling)
+  - [Example Components](#example-components)
+- [Installation](#installation)
+  - [CLI](#cli)
+  - [Manual](#manual)
+- [Commands](#commands)
+- [Demo Implementation](#demo-implementation)
+
+## Overview
+
+### Tech Stack
+
+- [SolidStart](https://start.solidjs.com/getting-started/what-is-solidstart/)
+- [TanStack Query](https://tanstack.com/query/v4/docs/solid/overview)
+- [Tailwind CSS](https://tailwindcss.com/)
+
+### Included Tooling
+
+- [Vitest](https://vitest.dev/) - Test runner
+- [Storybook](https://storybook.js.org/) - Component library
+- [Typescript](https://www.typescriptlang.org/) - Type checking
+<!-- - [ESLint](https://eslint.org/) - Code linting -->
+<!-- - [Prettier](https://prettier.io/) - Code formatting -->
+
+### Example Components
+
+#### Counter `src/components/CounterExample`
+
+A small counter example that showcases how to use state in SolidJS. You can view the example on the `src/pages/Counter`.
+
+Signals can live outside of components. Each relevant component subscribes to its value by using it.
 
-## Developing
+#### Fetch `src/components/FetchExample`
 
-Once you've created a project and installed dependencies with `npm install` (or `pnpm install` or `yarn`), start a development server:
+Demonstrates how to retrieve data from a third-party API by using `createResource`.
+
+## Installation
+
+### CLI (Recommended)
 
 ```bash
-npm run dev
+npm create @this-dot/starter --kit solidstart-tanstackquery-tailwind-modules
+```
+
+- Follow the prompts to select the solidstart-tanstackquery-tailwind-modules starter kit and name your new project.
+- `cd` into your project directory and run `npm install`.
+- Run `npm run dev` to start the development server.
+- Open your browser to `http://localhost:3000` to see the included example code running.
+
+### Manual
 
-# or start the server and open the app in a new browser tab
-npm run dev -- --open
+```bash
+git clone https://github.com/thisdot/starter.dev.git
 ```
 
-## Building
+- Copy and rename the `starters/solidstart-tanstackquery-tailwind-modules` directory to the name of your new project.
+- `cd` into your project directory and run `pnpm install`.
```

Comment:
Since we made the example above with `npm` and also the command in the line below is with `npm`, we should use `npm` also here instead of `pnpm`